### PR TITLE
added navbar

### LINKED ISSUE #2: added navbar
build navbar component

### PR DESCRIPTION
**Pull request by mariyasada**
nav
added navbar

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
 body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  width: 100%;
 }
 
 code {
```
Comment (tusharkandpal) [inline src/index.css]: Can remove this `code` tag. If not in use

**Review by tusharkandpal — CHANGES_REQUESTED** [review]
Looks good!

File: src/index.css (hunk 1 above)
Comment (mariyasada) [reply src/index.css]: Sure

**Review by tusharkandpal — APPROVED** [review]

**Review by Raghav888 — APPROVED** [review]
Approved

**Pull request merged**